CMakeLists.txt

```diff
@@ -177,7 +177,14 @@ install(DIRECTORY ${CESIUM_NATIVE_SPDLOG_INCLUDE_DIR}/spdlog DESTINATION "${CMAK
 
 install(TARGETS ${CESIUM_NATIVE_DRACO_LIBRARY})
 
-install(TARGETS sqlite3)
+#if we're aliased then someone else has a sqlite3 lib for us to use, otherwise install our own
+get_target_property(_aliased sqlite3 ALIASED_TARGET)
+if(NOT _aliased)
+    message("sqlite3 not aliased, using ours in extern/sqlite3")
```

Comment:
I don't think we need a message in this normal case.

Reply:
I'm curious, too: why is it a problem to install the targets from an aliased sqlite3? Why does sqlite3 needs this special code, when other libraries used by both cesium-native and another project (such as Draco?) don't need this?